packages/nds-page-header/scss/page-header.scss

```diff
@@ -25,6 +25,12 @@
 .page-header {
   margin: rem(0 0 $spacing-x-large 0);
 
+  &__pre-heading {
+    display: block;
+    margin: rem(0);
+    @include font-size(h5);
```

Comment:
Could you move the mixin first to follow the convention that's elsewhere please?